gcloud-java-dns/src/test/java/com/google/gcloud/dns/DnsBatchTest.java

```diff
@@ -195,6 +230,10 @@ public void testCreateZone() {
 
   @Test
   public void testCreateZoneWithOptions() {
+    EasyMock.verify(dns);
+    EasyMock.reset(dns);
+    EasyMock.expect(dns.options()).andReturn(optionsMock);
+    EasyMock.replay(dns);
```

Comment:
I believe that also here we should test that the callback correctly forwards the result/exception.